### ISSUE TITLE
v0.226.0 has broken deploy for environment dependencies

### ISSUE THREAD

**Issue by sanjevraj** [databricks/setup-cli#124]
Version v0.226.0 fails when the resource file has a definition in job like:

```
      environments:
        - environment_key: ext_data-default
          spec:
            client: "1"
            dependencies:
              - beautifulsoup4==4.12.3      
``` 

Get the following error in github action:
```
Error: file doesn't exist ./resources/beautifulsoup4==4.12.3
``` 

This worked fine in version v0.225.0.

**Comment by pietern** [comment]
Thanks for reporting. We'll figure out a fix shortly.

**Comment by pietern** [comment]
The fix has been merged and will be part of the next release (v0.227.0) happening tomorrow.
